src/System.AppContext/tests/AppContext.Switch.Validation.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit;
+
```

Comment:
Nit: our standard is for tests to be in the same name + ".Tests" of the type they're testing, e.g. "System.Tests".  Not all of our tests follow this, but that's what we're moving to.